### DISCUSSION TITLE
Problema para contestar el cuestionario

### DISCUSSION THREAD

**Discussion by Ing-JuanMata** [LaunchX-InnovaccionVirtual/CursoIntroPython#235] [Q&A] [answered]
Alguien tiene problemas para poder ver o contestar el cuestionario?
![image](https://user-images.githubusercontent.com/60631492/153911276-092696bd-fc70-4ebf-99ab-fdbfb018a0a9.png)

**Comment by Dfredude** [comment]
Recuerda que lo habilitarán a medio día

**Reply by FernandaOchoa** [reply]
Está habilitado desde la madrugada del día de hoy, ingresen en módo privado u incógnito para no tener problemas.

**Comment by doguedogue** [comment, marked as answer]
Al inicio te pide usar tu cuenta de registro, si tienes alguna sesión de outlook personal, cierra sesión para  que te la pida.

**Asegúrate de ingresar con tu cuenta launchx de innovaccion.mx, ya que de lo contrario tu registro será inválido**

**Reply by FernandaOchoa** [reply]
Así es, a quienes les aparece de esa manera es porque únicamente funciona con tu cuenta de Innovacción. 

Mi recomendación es abrir el enlace en una pestaña de Incógnito, de lo contrario no podrás verlo y está activo desde las 00:30 horas, es decir desde la madrugada del día de hoy comenzó a funcionar.
